build/dependencies.props

```diff
@@ -1,6 +1,6 @@
 ﻿<Project>
   <PropertyGroup>
-    <AspNetCoreVersion>2.0.0-*</AspNetCoreVersion>
+    <AspNetCoreVersion>$(VersionPrefix)-*</AspNetCoreVersion>
```

Comment:
This causes an error:
```
C:\Users\appveyor\.dotnet\x64\sdk\2.0.0-preview3-006841\NuGet.targets(102,5): error : '-*' is not a valid version string.
```
VersionPrefix isn't yet defined at this point in the sample projects